### PR TITLE
added api health route, giving a 200

### LINKED ISSUE #150: Create a healthcheck endpoint like /api/health or /health to hook into various schedulers like marathon.
It'd would be really nice to have a REST endpoint like /api/health that responds with 200 OK at the very least.

### LINKED ISSUE #322: Ping route without authentification
Hi,

I need a ping route to make sure my load balancer see my Kafka Manager healthy, even though it's protected by username password. I've tried adding a `/ping` route, using the following code:

```
package controllers

import play.api.i18n.{I18nSupport, MessagesApi}
import play.api.mvc._

class Ping(val messagesApi: MessagesApi) extends Controller with I18nSupport {

  def ping = Action {
    Ok("pong")
  }

}
```

But I still can't access the route without authentication... 

How can I proceed?

### PR DESCRIPTION
**Pull request by simplesteph**
This fixes #150 and this fixes #322

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by simplesteph** [comment]
I have agreed to the CLA

**Comment by simplesteph** [comment]
@patelh how do i trigger the CI again ? I've signed the cla

**Comment by jamescarr** [comment]
retest

File: conf/application.conf [hunk 1]
```diff
@@ -36,6 +36,6 @@ basicAuthentication.enabled=false
 basicAuthentication.username="admin"
 basicAuthentication.password="password"
 basicAuthentication.realm="Kafka-Manager"
-
+basicAuthentication.excluded=["/api/health"] # ping the health of your instance without authentification
```
Comment (patelh) [inline conf/application.conf]: is there support for this already?

Comment (simplesteph) [inline conf/application.conf]: yes

Comment (simplesteph) [inline conf/application.conf]: https://github.com/yahoo/kafka-manager/blob/97329cc8bf462723232ee73dc6702c064b5908eb/app/controllers/BasicAuthenticationFilter.scala

**Pull request merged**